karma.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: karma
+  version: "0.120"
+  description: "A dashboard for managing alerts from Alertmanager"
+  epoch: 0
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+      - npm
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/prymitive/karma.git
+      tag: v${{package.version}}
+      expected-commit: 8009f4aa581980654e5c67b26b41e85d9e7ccce0
+
+  - name: install-npm-dependencies
+    working-directory: ui
+    pipeline:
+      - runs: |
+          mkdir -p node_modules/.bin
+          touch node_modules/.install
+          npm install --package-lock-only
+          npm ci
+          cat node_modules/bootstrap/scss/_root.scss | sed s/':root {'/'* {'/ > src/Styles/BootstrapRoot.scss
+          npm run build
+
+  - name: Copy build files
+    runs: |
+      cp -r ui/src ./ui/dist ./ui/mock ./ui/embed.go .
+
+  - uses: go/build
+    with:
+      modroot: cmd/${{package.name}}
+      packages: .
+      ldflags: -X main.version=${{package.version}}
+      output: ${{package.name}}
+
+test:
+  pipeline:
+    - name: Run Karma test
+      runs: |
+        karma --version
+    - name: "Check karma status"
+      uses: test/daemon-check-output
+      with:
+        start: "${{package.name}}"
+        timeout: 60
+        expected_output: |
+          Starting HTTP server
```

Comment:
If there is endpoint, can we do a simple `curl | grep` test here in the `post:`?

Reply:
Done. Added a check to test `/health` endpoint